app/services/iiif_print/derivative_rodeo_service_decorator.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# OVERRIDE IIIF Print v1.0.0 to only run thumbnail derivatives on thumbnail files
+
+module IiifPrint
+  module DerivativeRodeoServiceDecorator
+    def create_derivatives(filename)
+      if filename.downcase.ends_with?(Hyku::THUMBNAIL_FILE_SUFFIX)
+        named_derivatives_and_generators.delete_if { |named_derivative| named_derivative != :thumbnail }
```

Comment:
I'm wanting to check that this deletion does not alter all future values of `named_derivatives_and_generators`.  I think it should be okay.

This is a case where a test would be wonderful.

Reply:
After you've deleted the instance from `named_derivatives_and_generators` what is the value of `#named_derivatives_and_generators_by_type` and `.named_derivatives_and_generators_by_type`?

Reply:
> I'm wanting to check that this deletion does not alter all future values of `named_derivatives_and_generators`.  I think it should be okay.
> 
> This is a case where a test would be wonderful.

Good point, I'll do a check and see what I can do about a test